guides/RELEASE-PROCESS.md

```diff
@@ -1,76 +1,53 @@
 # Release process
 
-1. **Create a release draft on GitHub**
-
-   1. [Draft a new release on GitHub](https://github.com/CMSgov/design-system/releases/new)
-   1. Tag the release using the [SemVer specification release format](#versioning). For example, `v1.1.0`
-   1. Title the release using the release number. For example, `1.1.0`
-   1. Follow the format below for the release notes
-
-   ```
-   ## 🚨 Breaking/Behavioral changes
-   ## 🚀 Added
-   ## 💅 Changed
-   ## 🛠 Fixed
-   ## 📦 Internal
-   ## 🚫 Deprecated
-   ```
+1. **Publish to NPM**
 
-   **Note**: View commits since the last release on github by going to the [releases page](https://github.com/CMSgov/design-system/releases) or run: `` git log `git describe --tags --abbrev=0`..HEAD --oneline ``
+   1. Ensure you are logged into NPM. Check your user account with `npm whoami`.
 
-1. **Create a release commit**
+      **Note**: Your NPM account must access to the CMS group and provide a valid access token. To add a token, edit your `~/.npmrc` file so the contents are `//registry.npmjs.org/:_authToken={token}`
 
    1. Checkout the latest `master` branch and ensure that you don't have any local changes. If you do have local changes [stash or discard](https://docs.gitlab.com/ee/topics/git/numerous_undo_possibilities_in_git/#quickly-save-local-changes) them before going to the next step.
```

Comment:
I would maybe move `Checkout the latest master` to be the first step 1 under publish to npm